examples/gno.land/p/gov/dao/proposal.gno

```diff
@@ -0,0 +1,47 @@
+package dao
+
+import "std"
+
+type Status string
+
+// ACTIVE -> ACCEPTED -> EXECUTED
+// ACTIVE -> NOT ACCEPTED
+var (
+	Active      Status = "active"       // proposal is still active
+	Accepted    Status = "accepted"     // proposal gathered quorum
+	NotAccepted Status = "not accepted" // proposal failed to gather quorum
+	Executed    Status = "executed"     // proposal is executed
```

Comment:
Timedout

Reply:
Freeze?

Reply:
sorry, just noticed that it was the "DAO Status"

please rename the type DAOStatus

Reply:
oh no, not sorry; please, keep timedout, and freeze, but rename ProposalStatus

Reply:
+ add ExecutedError

Reply:
Renamed, and added:


794a353